.github/workflows/ci-test.yml

```diff
@@ -57,4 +57,9 @@ jobs:
 
       - name: Build with Hugo ${{ matrix.hugo }}
         working-directory: exampleSite
-        run: HUGO_THEME="Binario" hugo --themesDir ../.. --i18n-warnings -v
+        run: |
```

Comment:
What do you think about `sort -V`? I have no preference for this, but it should potentially look cleaner (not tested, I could be wrong). Just a suggestion.